install/migrations/update_10.0.x_to_11.0.0/form.php

```diff
@@ -401,3 +408,36 @@
     DAY_TIMESTAMP,
     param: 7
 );
+
+// Data format for ValidationField has changed
+if ($DB->tableExists('glpi_forms_destinations_formdestinations')) {
```

Comment:
I would expect some kind of check to make sure this is only executed once (checking the format ?), is this safe to run this multiple times?